_pytest/fixtures.py

```diff
@@ -23,6 +23,8 @@
 )
 from _pytest.outcomes import fail, TEST_OUTCOME
 
+PseudoFixtureDef = namedtuple('PseudoFixtureDef', ('cached_result', 'scope'))
```

Comment:
please use a frozen attrs class here,

Reply:
Hi @RonnyPfannschmidt , thanks for the feedback! Can you elaborate on the use of a frozen attr class? Do you mean using a class with `__slots__` attribute set?